### DISCUSSION TITLE
Welcome to Disease.sh API Discussions!

### DISCUSSION THREAD

**Discussion by ebwinters** [NovelCOVID/API#983] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by nibble0101** [comment]
Hi there!
My name is Joseph a.k.a @nibble0101. I came across this community when I was looking for an API for COVID-19 datasets.   Since then I have remained here trying to make the API better than I found it. I am always looking for the slightest opportunity to make contribution. Since I am a beginner in node, express and APIs in general, it is also a great opportunity to learn. 
Great to meet you all!

**Comment by nnguyen52** [comment]
https://disease.sh/v3/covid-19/continents/Australia doesnt work ?

**Reply by ebwinters** [reply]
> https://disease.sh/v3/covid-19/continents/Australia doesnt work ?

Please file a bug. The continent is Oceania anyways but that does not work either

**Comment by tjaska** [comment]
Hi! I was wondering why the vacciations API route
`router.get('/v3/covid-19/vaccine/coverage/countries', async (req, res) => {`
doesn't return country iso code. It seems the source data (`https://covid.ourworldindata.org/data/vaccinations/vaccinations.csv`) has it. I'm forced to use the souce data if i want to do any further processing since matching countries by their name is a pain.

**Reply by ebwinters** [reply]
There's no reason ISO is not returned. If you want to file a feature bug, someone might pick it up, or you're welcome to implement it yourself! It should not be very difficult since the source already has it

**Comment by lnhtanh99** [comment]
Hi!
The data of recovered since 8/5/21 return null on this api 'https://disease.sh/v3/covid-19/historical/all?lastdays=all'

**Comment by andrew29williams** [comment]
Hi Joseph! 👋
Welcome aboard! My name is Andrew Williams — great to meet you. I really respect your goal of making meaningful contributions while learning. That’s the spirit that keeps API communities thriving. I work with API development and security too, and if you ever want to explore some helpful tools or guides around APIs, feel free to check out my site: [Ficode](https://www.ficode.com/services/third-party-api-development). Looking forward to seeing what you build with the Disease.sh API!
